python/fapolicy_analyzer/ui/confirm_changes_dialog.py

```diff
@@ -0,0 +1,83 @@
+import gi
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
+import ui.strings as strings
+
+
+class ConfirmInfoDialog(Gtk.Dialog):
```

Comment:
For consistency the class name should correspond with the python file name.

Reply:
Done. Will push shortly.